Extensions/ShareTo/Images.xcassets/AddToReadingList.imageset/Contents.json

```diff
@@ -1,7 +1,7 @@
 {
   "images" : [
     {
-      "filename" : "reader_add.pdf",
+      "filename" : "Outline icons.pdf",
```

Comment:
Just wondering, why naming all pdf "Outline icons"?

Reply:
they are named like this in figma and thus have this name after export. I opted out of renaming the files for efficiency reasons

Reply:
Makes sense, it becomes kind of tedious, good idea 👍